### PR TITLE
Send mail participants

### PR DESCRIPTION
**Pull request by quanttn-0861**
Peek 2019-04-22 11-38

### CODE REVIEW

File: app/Jobs/jobSendInvitationParticipants.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+namespace App\Jobs;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Support\Facades\Mail;
+use App\Mail\SendInvitationParticipants;
```
Comment (framgiaciv3) [inline app/Jobs/jobSendInvitationParticipants.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Classes.ValidClassName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Jobs/jobSendInvitationParticipants.php</br>**Line**: 13</br>Class name "jobSendInvitationParticipants" is not in PascalCase format

Comment (framgiaciv3) [inline app/Jobs/jobSendInvitationParticipants.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Classes.ValidClassName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Jobs/jobSendInvitationParticipants.php</br>**Line**: 13</br>Class name "jobSendInvitationParticipants" is not in PascalCase format

File: app/Http/Controllers/PollController.php [hunk 2]
```diff
@@ -98,6 +101,14 @@ public function store(Request $request)
                 Setting::create($inputSetPassword);
             }
 
+            $tagsMail = $request->tagsEmail;
+
+            if (is_array($tagsMail)) {
+                foreach($tagsMail as $value) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/PollController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Controllers/PollController.php</br>**Line**: 107</br>Expected 1 space(s) after FOREACH keyword; 0 found

File: app/Jobs/jobSendInvitationParticipants.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+use App\Http\Controllers\PollController;
```
Comment (framgiaciv3) [inline app/Jobs/jobSendInvitationParticipants.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Classes.ValidClassName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Jobs/jobSendInvitationParticipants.php</br>**Line**: 14</br>Class name "jobSendInvitationParticipants" is not in PascalCase format

File: app/Jobs/JobSendInvitationParticipants.php [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        $this->email = $email;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
```
Comment (framgiaciv3) [inline app/Jobs/JobSendInvitationParticipants.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Jobs/JobSendInvitationParticipants.php</br>**Line**: 37</br>String "Start send email" does not require double quotes; use single quotes instead

File: app/Jobs/JobSendInvitationParticipants.php [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        echo "Start send email";
+
+        Mail::to($this->email)->send(new SendInvitationParticipants());
```
Comment (framgiaciv3) [inline app/Jobs/JobSendInvitationParticipants.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Jobs/JobSendInvitationParticipants.php</br>**Line**: 41</br>String "End send mail" does not require double quotes; use single quotes instead

File: resources/views/mailer/sendInvitationParticipants.blade.php [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Test Email</title>
+</head>
+<body>
+    <h1>Test Email</h1>
```
Comment (tranducquoc) [inline resources/views/mailer/sendInvitationParticipants.blade.php]: sao không làm luôn mà để như này e?

Comment (quanttn-0861) [reply resources/views/mailer/sendInvitationParticipants.blade.php]: dạ tại chưa có làm phần Link anh, sau pull này là pull link em mới bỏ hết cả cái giao diện vô để hiển thị cho gửi mail đó anh :D

**Pull request merged**